### DISCUSSION TITLE
Assumption of linearity

### DISCUSSION THREAD

**Discussion by juandavidgutier** [DoubleML/doubleml-for-r#176] [Q&A]
Hi developers team,

I have a question about the models IRM and IIVM do they both impose the assumption that the outcome is a linear function of the treatment, or they are fully flexible non-parametric estimators of the relationship between the treatment and the outcome? And by the name, I assume that the models PLR and PLIV impose this linear assumption.

Thanks a lot for your answer.

**Comment by PhilippBach** [comment]
see reply here: https://github.com/DoubleML/doubleml-for-py/discussions/176

**Comment by PhilippBach** [comment]
I lock this conversation because it is redundant to the one opened in the [Python repository](https://github.com/DoubleML/doubleml-for-py/discussions/176). 

I'd encourage everyone interested to comment and discuss the question there 😄
